cmd/kube-controller-manager/app/plugins_storage_minimal.go

```diff
@@ -0,0 +1,152 @@
+// +build skip_storage_plugins_requiring_vendor_api
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	// This file exists to force the desired plugin implementations to be linked.
+	// This should probably be part of some configuration fed into the build for a
+	// given binary target.
+
+	"fmt"
+	"k8s.io/klog/v2"
+
+	// Cloud providers
+	cloudprovider "k8s.io/cloud-provider"
+	// Volume plugins
```

Comment:
rather than trying to mentally diff imports between two files and keep Probe*Plugins methods in sync, this might be easier to follow if there were three files:

* cloud provider storage
* vendor storage
* non-vendor storage


each could have a build tag, add the relevant imports, and add functions to a list executed for each of the `Probe*Plugins` methods